src/reducers/index.js

```diff
@@ -1,21 +1,20 @@
 import { INCREMENT, DECREMENT } from '../actions';
 
-const initialState = {
-  count: 0
-}
-
+//REDUCER
 // Our reducer that handles our two action cases:
 // increment and decrement. It receives the state
 // of our redux store, along with an action created
 // by our action creator. What does the reducer
 // need to do with the count in each case?
-export default (state = initialState, action) => {
+export default (count = 0, action) => {
   switch (action.type) {
     case INCREMENT:
     // Fill in the body of this case
+    return ++count;
     case DECREMENT:
     // Fill in the body of this case
+    return --count;
     default:
-      return state;
+    count
```

Comment:
Nice job on the project. Everything is hooked up properly and functional. There should be `return` in line 18 of the reducers/index.js file